jwst/stpipe/step.py

```diff
@@ -707,9 +708,14 @@ def get_config_from_reference(cls, dataset, observatory=None):
         # Get the root logger, since the following operations
         # are happening in the surrounding architecture.
         logger = log.delegator.log
+        pars_model = cls.get_pars_model()
+
+        # Check if retrieval should be attempted.
+        if get_disable_crds_steppars():
```

Comment:
Shouldn't the arg be supplied here so that users can modify from the API? Or was it intentional to only get the info from the environment variable in this method?

Reply:
This would have required a bit of refractor upstream, which, for this functionality, was deemed not necessary. If it turns out to be an in-demand item, will consider it then.